### PR TITLE
Sync fonts_public.proto with internal

### PR DESCRIPTION
**Pull request by simoncozens**
This gives us the minisite_url field, making sure we don't overwrite it when updating things with the packager.

### CODE REVIEW

File: Lib/gftools/fonts_public.proto [hunk 1]
```diff
@@ -81,55 +84,6 @@ message FamilyFallbackProto {
   // Next = 6
 }
 
-//
```
Comment (m4rc1e) [inline Lib/gftools/fonts_public.proto:84]: I wonder if it's safe to strip this out? don't some Noto projects depend on these fields? e,g https://github.com/google/fonts/blob/b17d6bf90cadd249bb7a5135f3d76c23a2c89663/ofl/notoserifethiopic/METADATA.pb#L34

Comment (m4rc1e) [reply Lib/gftools/fonts_public.proto:84]: I'd be more comfortable if we just added the `minisite_url` field only.

Comment (m4rc1e) [reply Lib/gftools/fonts_public.proto:84]: Argh, it didn't show the block I highlighted. My concern is you're also deleting all the lang metadata fields when I suspect they may be used for Noto.

Comment (simoncozens) [reply Lib/gftools/fonts_public.proto:84]: No, what you're looking at is field 14 (`repeated string languages`).

I feel like it's better to minimise the changes between our copy and theirs, and theirs doesn't have the lang metadata (it belongs in the lang repo).

**Pull request merged**